analyses/molecular-subtyping-pathology/results/compiled_molecular_subtypes.tsv

```diff
@@ -1880,27 +1882,24 @@ PT_18QG3Z4H	7316-1052	BS_SPV8AHH5	Diffuse astrocytic and oligodendroglial tumor
-PT_19GCSK2S	7316-7622	BS_GXJ34W0F	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28	DMG, H3 K28	Updated via OpenPedCan subtyping
-PT_19GCSK2S	7316-7622	BS_YVWA1G4P	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28	DMG, H3 K28	Updated via OpenPedCan subtyping
-PT_19GCSK2S	7316-7622	BS_YZX742PQ	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28	DMG, H3 K28	Updated via OpenPedCan subtyping
+PT_19GCSK2S	7316-7622	BS_AH252MTC	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28, TP53	DMG, H3 K28	Updated via OpenPedCan subtyping
+PT_19GCSK2S	7316-7622	BS_FZ8D57HJ	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28, TP53	DMG, H3 K28	Updated via OpenPedCan subtyping
+PT_19GCSK2S	7316-7622	BS_GH6PH611	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28, TP53	DMG, H3 K28	Updated via OpenPedCan subtyping
+PT_19GCSK2S	7316-7622	BS_NXT37ZPA	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28, TP53	DMG, H3 K28	Updated via OpenPedCan subtyping
+PT_19GCSK2S	7316-7622	BS_E9PJZ7VC	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28, TP53	DMG, H3 K28	Updated via OpenPedCan subtyping
+PT_19GCSK2S	7316-7622	BS_GXJ34W0F	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28, TP53	DMG, H3 K28	Updated via OpenPedCan subtyping
+PT_19GCSK2S	7316-7622	BS_YVWA1G4P	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28, TP53	DMG, H3 K28	Updated via OpenPedCan subtyping
+PT_19GCSK2S	7316-7622	BS_YZX742PQ	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28, TP53	DMG, H3 K28	Updated via OpenPedCan subtyping
 PT_1A4HM6FR	7316-6980	BS_4FKXNJHW	Embryonal tumor	Medulloblastoma	Medulloblastoma, group 4	MB, Group4	MB, Group4	Subtype based on prediction;Updated via OpenPedCan subtyping
 PT_1A4HM6FR	7316-6980	BS_HC77B45S	Embryonal tumor	Medulloblastoma	Medulloblastoma, group 4	MB, Group4	MB, Group4	Subtype based on prediction;Updated via OpenPedCan subtyping
-PT_1AAYYGGY	7316-4996	BS_DRVEFVQ5	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28	NA	Updated via OpenPedCan subtyping
-PT_1AAYYGGY	7316-4996	BS_Y96RP1HJ	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28	NA	Updated via OpenPedCan subtyping
+PT_1AAYYGGY	7316-4996	BS_DRVEFVQ5	Diffuse astrocytic and oligodendroglial tumor	HGAT	Diffuse midline glioma, H3 K28-altered	DMG, H3 K28, TP53	NA	Updated via OpenPedCan subtyping
```

Comment:
When I re-run the script, I also spotted change of TP53 status. Is that something we expected to see?

Reply:
hmm, this could be due to changes in the HGG module if we did not run pathology and integrate. Can you open a new PR, stacked on this, to examine the HGG module? I see a lot of discrepancies for duplicate subtypes in those.